docs/providers/vcair.md

```diff
@@ -0,0 +1,251 @@
+# vCloud Air Provider   
+
+Connectivity to your vCloud Air API endpoint and connectivity to the network the nodes are created on is required and is usually not enabled by default.
+It it also required that your nodes be able to download chef-client and reach your chef-server.
+
+## known issues:
+
+* ssh authentication happens via ssh and public_key is not available in initial images
+* Only CentOS64-64Bit images currently function, Ubuntu, CentOS63-* and 32BIT versions do not set passwords correctly
+
+## machine_options /  :ssh_options
+
+Setting the password is required as inital ssh key authentication is not available in vCloud Air.
+
+### :auth_methods
+
+Must be set to ```[ 'password' ]```
+
+### :password
+
+Set to something random and secure.
+
+### :user_known_hosts_file
+
+Set to ```/dev/null``` as we have no way to retrieve the ssh-host fingerprint before connecting.
+
+## machine_options / :bootstrap_options   
+
+### :memory
+
+in megabytes
+
+### :cpus
+
+count of virtual cpus
+
+###  :use_private_ip_for_ssh
+
+Necessary to find the right ip
+
+### :image_name
+
```

Comment:
What is this knife list supposed to show?  Do users need to take the `Name` output and supply it to chef-provisioning as the `:image_name`?  It would be nice to have an explanation here.